### ISSUE TITLE
How do I setup notifications on a semi-regular schedule?

### ISSUE THREAD

**Issue by Li357** [zo0r/react-native-push-notification#594]
Is there any way to schedule notifications for every single week_day_ during a certain timeframe (from date A to date B) or have some boolean value indicating which days not to schedule/send a notification? 

I'm currently wanting to notify a user based on non-uniform intervals throughout the day (every 40 minutes or 60 minutes depending on data I get from a fetch request). Is it plausible with `PushNotification.localNotificationSchedule` to schedule a few notifications throughout the day for every weekday that does meet certain criteria, or would I have to switch to native code?

**Comment by tronin** [comment]
You can specify the time between notifications in "repeatType" property. You need to define time in ms (please see docs)

Current version of library doesn't support repeatType for iOS, but you can pull this changes into your own fork https://github.com/zo0r/react-native-push-notification/pull/381/files

If you need different time in other cases you can set repeatType in your business logic when your app is running, but don't forget to reset old scheduled notifications.

**Comment by Li357** [comment]
I can use `'time'` for multiple notifications a day, thanks.

How could I selectively not schedule notifications on certain days even without opening the app to do a check with `repeatType: 'week'`? For example, not show notifications on a certain Monday because of a boolean value or stop/start notifications after some date?

**Comment by tronin** [comment]
You can specify the only time between notifications, not more. I think it is Apple API feature.

**Comment by yelkamel** [comment]
hello @Li357  have you done that finally ? I try to do the same notification only on week day or weedend depend on user setting

**Comment by Li357** [comment]
@yelkamel I have not. I resorted to scheduling up to 64 local notifications (ios limitation) as a background fetch/on app open. Not the best solution but it works for now. If you want something that goes off every single weekday/weekend without fail, `repeatInterval` should work for you (if you have less than 64 repeating notifications). I had to do something different because on certain weeks I didn't want them to go off.
